### PR TITLE
Task 42672: Make the default number of items per table configurable

### PR DESCRIPTION
**Pull request by Rdidmohamedelhadi**

### CODE REVIEW

File: analytics-webapps/src/main/webapp/WEB-INF/jsp/analytics-table.jsp [hunk 1]
```diff
@@ -17,7 +19,9 @@
 <portlet:actionURL var="saveSettingsURL" />
 
 <div class="VuetifyApp">
-    <% int generatedId = (int) (Math.random() * 1000000l); %>
+    <% int generatedId = (int) (Math.random() * 1000000l);
+    String limit = PropertyManager.getProperty("exo.analytics.table.default.limit");
```
Comment (boubaker) [inline analytics-webapps/src/main/webapp/WEB-INF/jsp/analytics-table.jsp]: we should add a property in `AnalyticsTableFilter` and a new Setting value in Table settings UI

File: analytics-webapps/src/main/webapp/vue-app/table-portlet/components/settings/AnalyticsTableSetting.vue [hunk 2]
```diff
@@ -10,7 +10,20 @@
     </template>
     <template slot="content">
       <v-card-text v-if="tableSettings">
+        <v-subheader class="px-0">
+        <h4 class="mt-0">{{ $t('analytics.limit') }}</h4>
```
Comment (boubaker) [inline analytics-webapps/src/main/webapp/vue-app/table-portlet/components/settings/AnalyticsTableSetting.vue]: formatting

File: analytics-webapps/src/main/webapp/vue-app/table-portlet/components/settings/AnalyticsTableSetting.vue [hunk 3]
```diff
@@ -10,7 +10,20 @@
...
+        <v-text-field
+          v-model="tableSettings.pageSize"
+          :label="$t('analytics.limit')"
+          :placeholder="$t('analytics.limit')"
+          class="pt-2"
+          type="number"
+          outlined
+          required />
+        <v-subheader class="px-0">
         <h4 class="mt-0">{{ $t('analytics.mainColumn') }}</h4>
```
Comment (boubaker) [inline analytics-webapps/src/main/webapp/vue-app/table-portlet/components/settings/AnalyticsTableSetting.vue]: formatting

Comment (Rdidmohamedelhadi) [reply analytics-webapps/src/main/webapp/vue-app/table-portlet/components/settings/AnalyticsTableSetting.vue]: done

**Review by boubaker — APPROVED** [review]

**Pull request merged**